config/routes.rb

```diff
@@ -1,4 +1,12 @@
 Rails.application.routes.draw do
+  get 'admin', to: 'admin#index'
+  controller :sessions do
+    get 'login' => :new
+    post 'login' => :create
+    delete 'logout' => :destroy
+  end
+  get 'admin/index'
```

Comment:
This would route the same as line 2 wouldn't it?

Reply:
ah yes, they make you duplicate routes then say delete them lol. Missed this one